README.md

```diff
@@ -1771,9 +1770,7 @@ Handy metrics based on numbers above:
 * [Microsoft Engineering](https://engineering.microsoft.com/)
 * [Microsoft Python Engineering](https://blogs.msdn.microsoft.com/pythonengineering/)
 * [Netflix Tech Blog](http://techblog.netflix.com/)
-* [Paypal Developer Blog](https://devblog.paypal.com/category/engineering/)
```

Comment:
Paypal moved their blog to medium https://medium.com/paypal-engineering, please update to this new link

Reply:
Updated